common/src/main/java/com/linecorp/line/auth/fido/fido2/common/mdsv3/metadata/ExtensionDescriptor.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2018 LINE Corporation. All rights reserved.
+ * LINE Corporation PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ */
+
+package com.linecorp.line.auth.fido.fido2.common.mdsv3.metadata;
+
+import lombok.Data;
+
+@Data
+public class ExtensionDescriptor {
```

Comment:
This structure is missing the `tag` field: https://fidoalliance.org/specs/mds/fido-metadata-statement-v3.0-ps-20210518.html#extensiondescriptor-dictionary

Reply:
Fixed in 17a944d.